Dockerfile-argo

```diff
@@ -0,0 +1,51 @@
+# Installs workflow-generator, kubectl and argo. Runs workflow-generator on the machine-config
```

Comment:
maybe use the opportunity to rename this to follow the same scheme as the other dockerfiles?